.travis.yml

```diff
@@ -10,7 +10,8 @@ jobs:
     include:
         - name: "Java 8"
           jdk: openjdk8
-          script: mvn clean install sonar:sonar -Dsonar.organization=apache -Dsonar.projectKey=apache_incubator-tamaya
+          script: mvn clean install
```

Comment:
Shouldn't we do a 
`mvn clean org.jacoco:jacoco-maven-plugin:prepare-agent install sonar:sonar`
to properly get all the code coverage and so on?

Reply:
The jacoco task will have already completed in the `script` stage (jacoco runs during the `verify` task). So long as we don't `mvn clean` during `after_success`, those data files generated by jacoco will be present and will be used by the `sonar:sonar` task. The best way to verify this would be to close the PR from this branch and re-open it in the apache repo (though it was useful to verify that the CI task succeeded here)

Reply:
CI works fine and did before :)